app/Filament/Resources/ServerResource/Pages/EditServer.php

```diff
@@ -659,7 +664,37 @@ public function form(Form $form): Form
                                     ->deletable(false)
                                     ->addable(false)
                                     ->columnSpan(4),
-                            ])->columns(4),
+                                Forms\Components\Actions::make([
+                                    Action::make('createDatabase')
```

Comment:
Hide the button, when the config option for creating databases is disabled?
`config('panel.client_features.databases.enabled')`